src/scenes/CreateEvent/components/DateRow.js

```diff
@@ -1,26 +1,19 @@
 import React from 'react';
-import { Grid, TextField, withStyles } from '@material-ui/core';
+import { Grid, Typography, withStyles } from '@material-ui/core';
+import { injectIntl } from 'react-intl';
 
 import { DateTimePickerCombo } from './DateTimePickerCombo';
 import styles from './styles';
 
-export const DateRow = withStyles(styles, { withTheme: true })(({ classes, error, blockNum, intl, isOpen, ...props }) => (
-  <Grid container>
+export const DateRow = withStyles(styles, { withTheme: true })(injectIntl(({ classes, intl, error, blockNum, isOpen, ...props }) => (
+  <Grid container direction="row" alignItems="center">
     <Grid item xs={12} sm={6}>
-      <DateTimePickerCombo
-        fullWidth
-        error={error}
-        {...props}
-      />
+      <DateTimePickerCombo fullWidth error={error} {...props} />
     </Grid>
-    <Grid item xs={12} sm={6}>
-      <TextField
-        fullWidth
-        disabled
-        InputProps={{ classes: { input: classes.createEventTextField } }}
-        placeholder="Block Number"
-        value={blockNum ? `Block: ${blockNum}` : ''}
-      />
+    <Grid item xs>
+      <Typography variant="body2">
+        {blockNum ? `~ ${intl.formatMessage({ id: 'str.block', defaultMessage: 'Block' })} ${blockNum}` : ''}
```

Comment:
do we need this `~`

Reply:
Yup. `~` means approximate. And we are approximating the future block number.

Reply:
got it.